pkg/channel/distributed/dispatcher/dispatcher/dispatcher.go

```diff
@@ -284,6 +284,14 @@ func (d *DispatcherImpl) ConfigChanged(configMap *v1.ConfigMap) Dispatcher {
 		// Some of the current config settings may not be overridden by the configmap (username, password, etc.)
 		kafkasarama.UpdateSaramaConfig(newConfig, d.SaramaConfig.ClientID, d.SaramaConfig.Net.SASL.User, d.SaramaConfig.Net.SASL.Password)
 
+		// Enable Sarama Logging If Specified In ConfigMap
+		if ekConfig, err := kafkasarama.LoadEventingKafkaSettings(configMap); err == nil && ekConfig != nil {
+			kafkasarama.EnableSaramaLogging(ekConfig.Kafka.EnableSaramaLogging)
+			d.Logger.Debug(fmt.Sprintf("Setting Sarama Logging to %v", ekConfig.Kafka.EnableSaramaLogging))
+		} else {
```

Comment:
same sprintf comment as in reconciler.go

Reply:
Will do; ReviewDog suggested as much as well